application/src/main/java/org/thingsboard/server/controller/BaseController.java

```diff
@@ -828,19 +825,14 @@ public static Exception toException(Throwable error) {
     }
 
     protected <E extends HasName> String entityToStr(E entity) {
-        try {
-            return json.writeValueAsString(json.valueToTree(entity));
-        } catch (JsonProcessingException e) {
-            log.warn("[{}] Failed to convert entity to string!", entity, e);
-        }
-        return null;
+        return JacksonUtil.toString(JacksonUtil.valueToTree(entity));
```

Comment:
You are affecting the behavior of the method. Before your change, this method never throws exceptions. After your change, you may receive the unchecked IllegalArgumentException. Please make sure you do not change the behavior of the code during refactoring and review all changes according to this comment.